rhoc/pom.xml

```diff
@@ -0,0 +1,151 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.drools.ruleops</groupId>
+    <artifactId>ruleops-parent</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>rhoc</artifactId>
```

Comment:
```suggestion
  <artifactId>ruleops-rhoc</artifactId>
```

?

Reply:
I'd rather keep "rules only" modules outside of RuleOps, eventually I think they should be moved to another project as well